src/main/java/uk/gov/hmcts/reform/em/stitching/pdf/PDFMerger.java

```diff
@@ -53,9 +56,9 @@ public File merge() throws IOException {
                 currentPageNumber += coverPageDocument.getNumberOfPages();
                 pdfOutline.addItem(0, "Cover Page");
             }
-          
+
             if (bundle.hasTableOfContents()) {
-                this.tableOfContents = new TableOfContents(document, bundle);
+                this.tableOfContents = new TableOfContents(document, bundle,documents);
```

Comment:
Formatting issue here, add a space between comma and next var

Reply:
done